src/components/displayAllComponent.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import ChooseFolder from "./chooseFolder/chooseFolder";
+import { NavLink,Redirect } from "react-router-dom";
+import SvgSettingOptions from "./svgSetting/svgSettingOptions";
+import "../components/checkBoxSelection/checkBoxSelection.scss";
+import { changeObj, optimizeSvg } from "../../src/components/functions";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import {
+  ADD_VARIABLE,
+  UPDATE_VARIABLE,
+  DELETE_VARIABLE
+} from "../store/actionTypes";
+
+class DisplayAllComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+     redirect : false
+    };
+  }
+  getOptimizeSvg = async () => {
+    let OriginalAndOptimised = {}
```

Comment:
start all variable names with lowercase please

Reply:
changed capital letters to camel case